roles/container-engine/cri-o/tasks/crio_repo.yml

```diff
@@ -1,5 +1,28 @@
 ---
 
+- block:
+    - name: Add Debian Backports apt repo
+      apt_repository:
+        repo: "deb http://deb.debian.org/debian {{ ansible_distribution_release }}-backports main"
+        state: present
+        filename: debian-backports
+      when:
+        - ansible_distribution == "Debian"
+        - ansible_distribution_version == "10"
```

Comment:
This line 9-11 seems redundant with line 22-24 in the same block.

Reply:
You're right @oomichi , I first add this condition on the 2 tasks, then put them in a block, and forgot to remove it.

Fixed, thanks !